charts/move2kube/README.md

```diff
@@ -24,8 +25,117 @@ The following table lists the configurable parameters of the Move2kube chart and
+oc -n ${M2K_INSTANCE_NS} adm policy add-scc-to-user anyuid -z default
+oc -n ${M2K_INSTANCE_NS} create secret generic sshkeys --from-file=id_rsa=${HOME}/.ssh/id_rsa --from-file=id_rsa.pub=${HOME}/.ssh/id_rsa.pub
+```
+
+Then force the pod to be recreated:
+```console
+oc -n ${M2K_INSTANCE_NS} scale deploy move2kube --replicas=0 && oc -n ${M2K_INSTANCE_NS} scale deploy move2kube --replicas=1
+```
+
+#### Set `M2K_ROUTE` and `BROKER_URL` for the Knative service
+As the Knative service cannot be updated, we need to delete if first and then re-create it with the helm command.
+
+Run the following command or follow the steps prompted at the end of the workflow installation to apply it to the `move2kubeURL` parameter:
+```console
+M2K_ROUTE=$(oc -n ${M2K_INSTANCE_NS} get routes move2kube-route -o yaml | yq -r .spec.host)
+oc -n ${TARGET_NS} delete ksvc m2k-save-transformation-func &&
+  helm upgrade move2kube orchestrator-workflows/move2kube -n ${TARGET_NS} --set workflow.move2kubeURL=https://${M2K_ROUTE}
+```
+
+#### Edit the `${WORKFLOW_NAME}-creds` Secret
+The token for sending notifications from the m2k workflow to RHDH notifications service needs to be provided to the workflow.
+
+Edit the secret `${WORKFLOW_NAME}-creds` and set the value of `NOTIFICATIONS_BEARER_TOKEN`:
+```
+WORKFLOW_NAME=m2k
+oc -n ${TARGET_NS} patch secret "${WORKFLOW_NAME}-creds" --type merge -p '{"data": { "NOTIFICATIONS_BEARER_TOKEN": "'$(oc get secrets -n rhdh-operator backstage-backend-auth-secret -o go-template='{{ .data.BACKEND_SECRET  }}')'"}}'
+```
+
+This secret is used in the `sonataflow` CR to inject the token as an environment variable that will be used by the workflow.
+
+Once the secret is updated, to have it applied, the pod shall be restarted. 
+Note that the modification of the secret does not currently restart the pod, the action shall be performed manually or, if you are following the next section, any change to the sonataflow CR will restart the pod.
+
+Note that when you run the `helm upgrade` command, the values of the secret are reseted.
+
+#### Set `M2K_ROUTE`, `K_SINK` and `BACKSTAGE_NOTIFICATIONS_URL` for the Sonataflow CR
+
+The value for `BACKSTAGE_NOTIFICATIONS_URL` in the command below is using the current default value, if the name of the backstage deployment or its namespace does not match, please update the value with the correct value from your cluster.
+
+Run the following to set `K_SINK`, `MOVE2KUBE_URL` and `BACKSTAGE_NOTIFICATIONS_URL`environment variable in the workflow:
+```console
+BACKSTAGE_NOTIFICATIONS_URL=http://backstage-backstage.rhdh-operator
+BROKER_URL=$(oc -n ${TARGET_NS} get broker -o yaml | yq -r .items[0].status.address.url)
+oc -n ${TARGET_NS} patch sonataflow m2k --type merge -p '{"spec": { "podTemplate": { "container": { "env": [{"name": "BACKSTAGE_NOTIFICATIONS_URL": "'${BACKSTAGE_NOTIFICATIONS_URL}'"},{"name": "K_SINK", "value": "'${BROKER_URL}'"}, {"name": "MOVE2KUBE_URL", "value": "https://'${M2K_ROUTE}'"}]}}}}'
```

Comment:
@gabriel-farache there is a type here, should be `{"name": "BACKSTAGE_NOTIFICATIONS_URL",  "value": "'${BACKSTAGE_NOTIFICATIONS_URL}'"}`

Reply:
Thx, the fix is part of https://github.com/parodos-dev/serverless-workflows-config/pull/846